.github/workflows/node.js.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
+        node-version: [18.x]
```

Comment:
do we need to change CI? I can see that there is still a step for node 16 build

Reply:
We should probably update/kill the buildkite